src/ape/managers/config.py

```diff
@@ -56,6 +56,9 @@ def __post_init__(self):
         if len(user_config.keys()) > 0:
             raise ConfigError("Unprocessed config items.")
 
+    def __repr__(self):
+        return "<ConfigManager>"
```

Comment:
Doing these reprs helps with the base `ape.__init__.py` related docs, especially when running in a local env